packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -180,8 +181,27 @@ class ThreadChannel extends BaseChannel {
       this.memberCount ??= null;
     }
 
+    if ('total_message_sent' in data) {
+      /**
+       * The number of messages ever sent in a thread, similar to `ThreadChannel#messageCount` except it
```

Comment:
Might as well link it!

```suggestion
       * The number of messages ever sent in a thread, similar to {@link ThreadChannel#messageCount} except it
```